opsdroid/core.py

```diff
@@ -349,7 +385,9 @@ def start_databases(self, databases):
                     self.eventloop.run_until_complete(database.connect())
 
     async def run_skill(self, skill, config, message):
-        """Execute a skill."""
+        """Execute a skill.
+        
```

Comment:
On this method you should add these Args and explain what they are/do. 
Since there is two lines with comment they should be inside the docstring as well